src/server/rdb_test.cc

```diff
@@ -668,15 +659,24 @@ TEST_F(RdbTest, LoadStream2) {
 }
 
 TEST_F(RdbTest, LoadStream3) {
-  auto ec = LoadRdb("RDB_TYPE_STREAM_LISTPACKS_2.rdb");
+  auto ec = LoadRdb("RDB_TYPE_STREAM_LISTPACKS_3.rdb");
```

Comment:
Nice catch!

Reply:
It was my bug, It looks I haven't saved it before commit when I wrote it